src/app/components/Sidebar/Sidebar.jsx

```diff
@@ -11,17 +11,17 @@ const fieldsetProps = {
   season: {
     fieldsetName: 'season',
     options: [
-      { name: '2018 Winter', value: '2018-01-01' },
-      { name: '2017 Fall', value: '2017-09-01' },
-      { name: '2017 Summer', value: '2017-06-01' },
-      { name: '2017 Spring', value: '2017-04-01' },
+      { name: '2018 Winter', value: '2018-01' },
+      { name: '2017 Fall', value: '2017-09' },
+      { name: '2017 Summer', value: '2017-06' },
+      { name: '2017 Spring', value: '2017-04' },
```

Comment:
Will we be dynamically generating this list from the `title` and list's `slug`?

Reply:
They are just temporary.  I will switch it to come from the endpoint once we have more data.